api/keystore/service.go

```diff
@@ -23,8 +23,14 @@ import (
 	jsoncodec "github.com/ava-labs/gecko/utils/json"
 )
 
+const (
+	// maxUserPassLen is the maximum length of the username or password allowed
+	maxUserPassLen = 1024
+)
+
 var (
-	errEmptyUsername = errors.New("username can't be the empty string")
+	errEmptyUsername     = errors.New("username can't be the empty string")
+	errUserPassMaxLength = errors.New(fmt.Sprintf("CreateUser call rejected due to username or password exceeding maximum length of %d chars", maxUserPassLen))
```

Comment:
Minor nitpick, but I think it would be cleaner for this to use `fmt.Errorf(...)` rather than `errors.New(fmt.Sprintf(...))`

Reply:
New commit has changed to fmt.Errorf()